### ISSUE TITLE
Different channels?

### ISSUE THREAD

**Issue by airlangga57** [Kitsunetic/head-project#1]
Hi Kitsunetic, 

I tried to run your code, but an error message comes up:
RuntimeError: Given groups=2, weight of size [64, 3, 7], expected input[256, 16, 126] to have 6 channels, but got 16 channels instead

**Comment by Kitsunetic** [comment]
Can you notify what network you used and what data as input?

For example, if you used network CRNNC, your input shape should be (BatchSize, SequenceLength, 6).
If not, you should edit either network's source code or dataset.

Thank you.

**Comment by airlangga57** [comment]
I tried to run CRNNC Hardswish by using the following command:
python run_training.py --window-size 120 --epochs 200 --batch-size 256 --cpus 8 --network CRNNC_Hardswish --dataset data/1220 --result results 

I just try to run your source code as it is. The raw data from gearVR, then I used the data generated by GenerateData.ipynb as input. 

So, in order to change the source code I just edit from head-project-FinalPresentation\networks\crnnc.py? Is this correct?
